test/feed-parser.test.js

```diff
@@ -0,0 +1,27 @@
+const feedParser = require('../src/feed-parser');
+
+/**
+ * feedParser(feed) async function returning an object if the feed URI is correct and there are
+ * things to parse. Invalid URI or non-feed URI should throw an error.
+*/
+test('passing a valid Atom feed URI and getting title of first post', async () => {
+  const data = await feedParser('https://c3ho.blogspot.com/feeds/posts/default/-/open-source?alt=rss');
```

Comment:
We can do this in the short term, but this kind of test is going to fail when you remove your blog at some point in the future.  A better option is to use a mock network request here, using something like https://www.npmjs.com/package/jest-fetch-mock.  We'd simulate getting a feed from the network, and return a prebaked XML document you can parse.

Reply:
Please file another issue to mock this out later.

Reply:
Filed issue under #210